docs/versioned_docs/version-2.1.0/data-sources/google.sheets.md

```diff
@@ -94,11 +94,13 @@ Use this operation delete a specific row from the sheet.
 | Fields      | description |
 | ----------- | ----------- |
 | Spreadsheet ID | It is mandatory to enter the spreadsheet-id. The spreadsheet-id can be found in the URL of the spreadsheet. Example URL: https://docs.google.com/spreadsheets/d/1W2S4re7zNaPk9vqv6_CqOpPdm_mDEqmLmzjVe7Nb9WM/edit#gid=0 - in this URL, the `1W2S4re7zNaPk9vqv6_CqOpPdm_mDEqmLmzjVe7Nb9WM` is the spreadsheet-id. |
+| [How to find google sheets id](https://fabform.io/a/how-to-find-google-sheet-id) |
```

Comment:
Can you please use the link to official documentation instead? https://developers.google.com/sheets/api/guides/concepts#spreadsheet